CondFormats/EcalObjects/src/EcalTPGTPMode.cc

```diff
@@ -0,0 +1,42 @@
+#include "CondFormats/EcalObjects/interface/EcalTPGTPMode.h"
+#include <iostream>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+EcalTPGTPMode::EcalTPGTPMode() {}
+
+EcalTPGTPMode::~EcalTPGTPMode() {}
+
+void EcalTPGTPMode::Print() const {
+  LogDebug("EcalTPGTPMode") << ">>> Trigger primitive mode:  ";
```

Comment:
Maybe it would be a better to define a print(std::ostream& out) function instead like for example here: https://github.com/cms-sw/cmssw/blob/master/CondFormats/EcalObjects/src/EcalMustacheSCParameters.cc#L33

That way one can then use it with cout or logDebug as desired.
Because like this you will get the header and footer of the logDebug message in every line.

Reply:
Header like this:
https://github.com/cms-sw/cmssw/blob/master/CondFormats/EcalObjects/interface/EcalMustacheSCParameters.h#L38-L43